terms/prime_or_finite_fields.md

```diff
@@ -13,6 +13,21 @@ defined as follows:
 
 In essence, all mathematical operations are performed modulo $p$.
 
+### Multiplicative Group
+
+For a informal explanation, $G$ is a list of powers of some "generator" value.
+
+For example, in the finite field $F_{17}$, if we choose the generator is
+$4$,  $G = \lbrace 1, 4, 16, 13 \rbrace$:
```

Comment:
I think it should be "if we choose the generator 4" or "if we choose 4 as the generator"